debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/RecordsStreamProducerIT.java

```diff
@@ -1103,6 +1103,32 @@ public void shouldReceiveChangesForTableWithoutPrimaryKey() throws Exception{
         assertRecordSchemaAndValues(expectedAfter, deletedRecord, Envelope.FieldName.AFTER);
     }
 
+    @Test()
+    public void testPassingStreamParams() throws Exception {
+        // Verify that passing stream parameters works by using the WAL2JSON add-tables parameter which acts as a
+        // whitelist.
+
+        PostgresConnectorConfig config = new PostgresConnectorConfig(TestHelper.defaultConfig()
+                .with(PostgresConnectorConfig.PLUGIN_NAME, "wal2json")
+                .with(PostgresConnectorConfig.STREAM_PARAMS, "add-tables=s1.should_stream")
+                .build());
+        setupRecordsProducer(config);
+        String statement = "CREATE SCHEMA s1;" +
+                "CREATE TABLE s1.should_stream (pk SERIAL, aa integer, PRIMARY KEY(pk));" +
+                "CREATE TABLE s1.should_not_stream (pk SERIAL, aa integer, PRIMARY KEY(pk));" +
+                "INSERT INTO s1.should_stream (aa) VALUES (123);" +
+                "INSERT INTO s1.should_not_stream (aa) VALUES (456);";
```

Comment:
Could you alter the order of the two `INSERT` statements? That way we'll be sure that the non-expected one has been processed by wal2json and excluded (instead of the theoretical possibility with the other order that the unexpected event actually _was_ created but we didn't consume it here in the test).

Reply:
Good idea, done.